server/cmwell-dc/src/main/scala/cmwell/dc/stream/DataCenterSyncManager.scala

```diff
@@ -481,6 +481,7 @@ class DataCenterSyncManager(dstServersVec: Vector[(String, Option[Int])],
               case JsDefined(JsArray(seq))
                 if seq.length == 1 && seq.head.isInstanceOf[JsString] =>
                 seq.head.as[String]
+              case _ => ""
```

Comment:
This should be Option (or any other way). Otherwise, it will add an empty modifier for infotons that doesn't have last-modified-by.